test/basic_test.cc

```diff
@@ -99,4 +99,12 @@ void BM_empty_stop_start(benchmark::State& state) {
 BENCHMARK(BM_empty_stop_start);
 BENCHMARK(BM_empty_stop_start)->ThreadPerCpu();
 
+template <class ...Args>
+void BM_with_args(benchmark::State& state, Args&&...) {
```

Comment:
C++11 language and library features are used elsewhere in the tests.  Only `cxx03_test.cc` compiles in C++03.

Is there a reason this particular feature needs guarding?